libs/utils/sidekick-decorate.js

```diff
@@ -4,7 +4,7 @@ const PUBLISH_BTN = '.publish.plugin button';
 const PROFILE = '.profile-email';
 const CONFIRM_MESSAGE = 'Are you sure? This will publish to production.';
 
-export default function stylePublish(sk) {
+function styleHelixPublish(sk) {
```

Comment:
nit: helix name is being phased out I believe in favor of AEM EDS I believe (even .hlx.page/live are supposed to change I think). Could leave as `stylePublish`?

Reply:
The default function is still called `stylePublish`. I renamed this one because it only applies to the old sidekick whose tag is  `<helix-sidekick>`, so I thought calling it `styleHelixPublish` made sense. (See lines 121-124)